public/data/food/ingredients.json

```diff
@@ -155,5 +155,13 @@
     "variants": {
       "organic": "a16fca295aff5ab9de57b3915cf8657a"
     }
+  },
+  {
+    "id": "mango",
+    "name": "Mangue",
+    "default": "c1e6f84a4fa89c2408bd7a2b2ed6cb96",
+    "default_origin": "OutOfEuropeAndMaghrebByPlane",
+    "raw_to_cooked_ratio": 0.856,
+    "variants": {}
   }
 ]
```

Comment:
Nit: would it be possible to have the scripts adding a trailing CR? Most editors add one when saving, that would prevent unecessary diffs I think.